pruner/example_programs/fabric/fabric.p4

```diff
@@ -46,9 +46,58 @@
 #include "include/int/int_main.p4"
 #endif // WITH_INT
 
+
+//////////////////
+
+#include <pna.p4>
+
+typedef bit<48>  EthernetAddress;
+
+
+
+struct headers_t {
+    ethernet_t ethernet;
+    ipv4_t ipv4;
+}
+
+struct main_metadata_t {
+   PortId_t dst_port;
+}
```

Comment:
Most of this should not be needed, just add a minimal function to make things compile.